CHANGELOG.md

```diff
@@ -74,6 +74,7 @@ Shipping methods can be removed by the user after it has been assigned to a chec
 - Add `search` to `giftCards` query - #13173 by @zedzior
 - Add `ProductBulkTranslate` mutation - #13329 by @SzymJ
 - Add `ProductVariantBulkTranslate` mutation - #13329 by @SzymJ
+- Add `metadata` to `AddressInput` field - #13248 by @Air-t
```

Comment:
Now as I read it I think that adding metadata to this input is the technical detail, while the significant information from the user point of view is that we added the ability to set public address metadata in checkout address mutation. So maybe something like this:
```
- Add the ability to set address public metadata in the following mutations: <mutation A>, <mutation B>, ... - #13248 by @Air-t
  - Add `metadata` to `AddressInput` field
```

Reply:
Listing the affected mutation will be also useful for QA team when they're testing it.